book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_5_make_it_move.md

```diff
@@ -63,7 +62,7 @@ There is some weird reason that messages published on Mac inside the container d
 
 For those that don't want or can't do the above where a window pops up, do the following (which will run directly on the robot): 
 
-    laptop $ dts duckiebot keyboard_control ![hostname] --cli --base_image duckietown/dt-core:daffy
+    laptop $ dts duckiebot keyboard_control ![hostname] --cli --base_image duckietown/dt-core
```

Comment:
let's take the `--base_image duckietown/dt-core` out completely since it is the default in the shell. This goes for everywhere

Reply:
This is now fixed.